railties/lib/rails/command.rb

```diff
@@ -89,6 +101,7 @@ def root
       end
 
       def print_commands # :nodoc:
+        puts "In addition to those commands, there are:\n"
```

Comment:
This has been moved out of the USAGE file because we don't need to show these commands outside a Rails application.